### PR TITLE
Do not check for expiry on captions change

### PR DESCRIPTION
**Pull request by MichaelKovalchuk**
Causes issues loading captions if the machine time is misconfigured

### CODE REVIEW

File: core/d2l-content-viewer/d2l-content-viewer.js [hunk 1]
```diff
@@ -218,22 +218,24 @@ class ContentViewer extends InternalLocalizeMixin(LitElement) {
 	}
 
 	async _onTrackLoadFailed() {
-		const elapsedTimeSinceLastTrackLoadFailed = (new Date()).getTime() - (this._lastTrackLoadFailedTime || 0);
-		const trackErrorFetchIntervalElapsed = elapsedTimeSinceLastTrackLoadFailed > TRACK_ERROR_FETCH_INTERVAL_MILLISECONDS;
+		const timeNow = Date.now();
+		const timeSinceLastTrackLoadFailed = timeNow - (this._lastTrackLoadFailedTime || 0);
```
Comment (MichaelKovalchuk) [inline core/d2l-content-viewer/d2l-content-viewer.js:222]: behaves the same as before. little less verbose

**Review by rcausey — APPROVED** [review]

File: core/d2l-content-viewer/d2l-content-viewer.js [hunk 2]
```diff
@@ -218,22 +218,24 @@ class ContentViewer extends InternalLocalizeMixin(LitElement) {
...
-			this._lastTrackLoadFailedTime = (new Date()).getTime();
-			this._trackErrorFetchTimeoutId = setTimeout(this._loadCaptions.bind(this), TRACK_ERROR_FETCH_INTERVAL_MILLISECONDS);
 		}
 	}
 
 	async _onTracksChanged() {
-		if (Date.now() > this._captionsSignedUrlExpireTime) {
-			await this._loadCaptions();
-		}
+		await this._loadCaptions();
```
Comment (qwangd2l) [inline core/d2l-content-viewer/d2l-content-viewer.js:238]: I'm just wondering but why do we not want to check expiry?

**Review by qwangd2l — APPROVED** [review]

File: core/d2l-content-viewer/d2l-content-viewer.js (hunk 2 above)
Comment (MichaelKovalchuk) [reply core/d2l-content-viewer/d2l-content-viewer.js:238]: if the machine time is misconfigured, it causes the issues with loading captions

Comment (MichaelKovalchuk) [reply core/d2l-content-viewer/d2l-content-viewer.js:238]: expiry here is a timestamp sent from the server

**Pull request merged**